src/components/productListing/Products.js

```diff
@@ -1,27 +1,29 @@
-import { useEffect , useState} from "react";
+import { useEffect } from "react";
 import { Link } from "react-router-dom";
 import { useCart } from "../../contexts/cart-context";
+import { useProductList } from "../../contexts/productlist-context";
 import { useWishlist } from "../../contexts/wishlist-context";
```

Comment:
you can create index file in context and access all context using one file

Reply:
Thanks for the suggestion. Currently I am not planning to create another file to organize the imports.